labgrid/factory.py

```diff
@@ -1,23 +1,28 @@
+import inspect
```

Comment:
s/not bound/not registered/ in the commit message ("bound" has a different meaning in labgrid).